tests/foreman/ui/test_org.py

```diff
@@ -671,35 +689,38 @@ def test_remove_medium(self, medium_name):
         """
         strategy, value = common_locators['entity_select']
         strategy1, value1 = common_locators['entity_deselect']
-        org_name = gen_string('alpha')
-        medium = entities.Media(
-            name=medium_name,
-            path_=INSTALL_MEDIUM_URL % gen_string('alpha', 6),
-            os_family='Redhat',
-        ).create()
-        self.assertEqual(medium.name, medium_name)
         with Session(self.browser) as session:
-            make_org(session, org_name=org_name,
-                     medias=[medium_name])
-            self.org.search(org_name).click()
-            session.nav.click(tab_locators['context.tab_media'])
-            element = session.nav.wait_until_element((strategy1,
-                                                      value1 % medium_name))
-            # Item is listed in 'Selected Items' list and not 'All Items' list.
-            self.assertIsNotNone(element)
-            self.navigator.go_to_org()
-            self.org.update(org_name, medias=[medium_name],
-                            new_medias=None)
-            self.org.search(org_name).click()
-            session.nav.click(tab_locators['context.tab_media'])
-            element = session.nav.wait_until_element((strategy,
-                                                      value % medium_name))
-            # Item is listed in 'All Items' list and not 'Selected Items' list.
-            self.assertIsNotNone(element)
+            for medium_name in generate_strings_list():
```

Comment:
Same comment as before about using one single organization and adding entity to it and asserting its presence. Then, remove entity and assert that it is gone from the organization. Not a blocker for this PR.